tests/unit/records/pandas/test_format_for_csv.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import unittest
+from mock import patch, call
+from records_mover.records.pandas import format_df_for_csv_output
+from records_mover.records.schema import RecordsSchema
+from records_mover.records import DelimitedRecordsFormat
+
+
+class TestFormatForCsv(unittest.TestCase):
+    def test_format_df_for_csv_output(self):
+        schema_data = {
+            'schema': "bltypes/v1",
+            'fields': {
+                "date": {
+                    "type": "date",
+                    "index": 1,
+                },
+                "time": {
+                    "type": "time",
+                    "index": 2,
+                },
+                # TODO:
+                # "timestamp": {
+                #     "type": "datetime",
+                #     "index": 3,
+                # },
+                # "timestamptz": {
+                #     "type": "datetimetz",
+                #     "index": 4,
+                # }
+            }
+        }
+        records_format = DelimitedRecordsFormat()
+        records_schema = RecordsSchema.from_data(schema_data)
+        data = {
+            'date': [pd.Timestamp(year=1970, month=1, day=1)],
+            # TODO: This is an ugly way to represent a time alone.
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker